Gems/Atom/Asset/Shader/Code/Source/Editor/ShaderBuildArgumentsManager.h

```diff
@@ -65,8 +65,8 @@ namespace AZ
             // The value of this registry key is customizable by the user.
             static constexpr char ConfigPathRegistryKey[] = "/O3DE/Atom/Shaders/Build/ConfigPath";
 
-            static constexpr char DefaultConfigPathDirectory[] = "@gemroot:AtomShader@/Config";
-            static constexpr char ShaderBuildOptionsJson[] = "shader_build_options.json";
+            static constexpr char DefaultConfigPathDirectory[] = "@gemroot:AtomShader@/Assets/Config/Shader";
```

Comment:
Given that these are not real assets, instead just a means to stablish dependencies it is very important to drop a README.md in this folder explaining why these files "appear" as assets. Otherwise, a year from now, nobody will remember why some settings files are in the assets folder.